customers-tomcat-legacy/README.md

```diff
@@ -0,0 +1,65 @@
+# Customer-tomcat-legacy application
+
+## Goals
+In this example, you will learn the insights on what changes are required to containerize a Java application running on tomcat server. This example uses Tackle operator to perform analysis on the application source code, the results of which produces insights needed to containerize the app.
```

Comment:
nit on s/tackle/konveyor

I realize we haven't yet renamed to Konveyor, just a question if we want to update this doc to consider Konveyor now, or delay that change to later.

Reply:
I thought about it too. I can keep a reminder to update all the references once the Konveyor renaming is official.

Reply:
I would just as soon update name on new things, because we know it is going to happen less work to clean it all up later IMO.

Reply:
done